demo/build.gradle

```diff
@@ -28,7 +29,8 @@ android {
 }
 
 dependencies {
-	implementation project(":lib")
+	// implementation project(":lib")
+	implementation 'com.github.okdroid:checkablechipview:0.0.9'
```

Comment:
This is for test right? I think we should revert it to ':lib' before promoting to everybody (at least I guess it's a common practice to use local module instead of remote one)

Reply:
Yes, will remove once it's on jcenter